### PR TITLE
Addition

### PR DESCRIPTION
**Pull request by CYB3R-G0D**
**Added**

1. Yalp store
2. monocles browser
3. Stylish calculator
4. Pulse music
5. ImagePipe
6. Trail Sense

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ Table of Contents
     - [**Fossdroid**](https://fossdroid.com/)
     - [**Foxy Droid**](https://github.com/kitsunyan/foxy-droid) <sup>**[[F-Droid](https://f-droid.org/app/nya.kitsunyan.foxydroid)]**</sup>
 - [**Aurora Store**](https://gitlab.com/AuroraOSS/AuroraStore) <sup>**[[F-Droid](https://f-droid.org/app/com.aurora.store)]**</sup>
+- [**Yalp Store**](https://github.com/yeriomin/YalpStore) <sup>**[[F-Droid](https://f-droid.org/en/packages/com.github.yeriomin.yalpstore/)]**</sup>
```
Comment (offa) [inline README.md:66]:
```suggestion
- [**Yalp Store**](https://github.com/yeriomin/YalpStore) <sup>**[[F-Droid](https://f-droid.org/app/com.github.yeriomin.yalpstore)]**</sup>
```

File: README.md [hunk 2]
```diff
@@ -92,11 +93,13 @@ Table of Contents
 - [**DuckDuckGo Privacy Browser**](https://github.com/duckduckgo/Android) <sup>**[[F-Droid](https://f-droid.org/app/com.duckduckgo.mobile.android)]**</sup>
 - [**SmartCookieWeb**](https://github.com/CookieJarApps/SmartCookieWeb) <sup>**[[F-Droid](https://f-droid.org/app/com.cookiegames.smartcookie)]**</sup>
 - [**Privacy Browser**](https://gitweb.stoutner.com/?p=PrivacyBrowser.git) <sup>**[[F-Droid](https://f-droid.org/app/com.stoutner.privacybrowser.standard)]**</sup>
+- [**monocles browser**](https://codeberg.org/Arne/monocles_browser) <sup>**[[F-Droid](https://f-droid.org/en/packages/de.monocles.browser)]**</sup>
```
Comment (offa) [inline README.md:96]:
```suggestion
- [**monocles browser**](https://codeberg.org/Arne/monocles_browser) <sup>**[[F-Droid](https://f-droid.org/app/de.monocles.browser)]**</sup>
```

File: README.md [hunk 3]
```diff
@@ -92,11 +93,13 @@ Table of Contents
...
 
 
 ### • Calculator
 - [**Calculator**](https://github.com/Xlythe/Calculator) <sup>**[[F-Droid](https://f-droid.org/app/com.xlythe.calculator.material)]**</sup>
 - [**Simple Calculator**](https://github.com/SimpleMobileTools/Simple-Calculator) <sup>**[[F-Droid](https://f-droid.org/app/com.simplemobiletools.calculator)]**</sup>
+- [**Stylish calculator**](https://github.com/jchmrt/clean-calculator) <sup>**[[F-Droid](https://f-droid.org/en/packages/home.jmstudios.calc)]**</sup>
```
Comment (offa) [inline README.md:102]:
```suggestion
- [**Stylish Calculator**](https://github.com/jchmrt/clean-calculator) <sup>**[[F-Droid](https://f-droid.org/app/home.jmstudios.calc)]**</sup>
```

File: README.md [hunk 4]
```diff
@@ -247,6 +250,8 @@ Table of Contents
 - [**KouChat**](https://github.com/blurpy/kouchat-android) <sup>**[[F-Droid](https://f-droid.org/app/net.usikkert.kouchat.android)]**</sup>
 - [**Feeder**](https://gitlab.com/spacecowboy/Feeder) <sup>**[[F-Droid](https://f-droid.org/app/com.nononsenseapps.feeder)]**</sup>
 - [**Scrambled Exif**](https://gitlab.com/juanitobananas/scrambled-exif) <sup>**[[F-Droid](https://f-droid.org/app/com.jarsilio.android.scrambledeggsif)]**</sup>
+- [**ImagePipe**](https://codeberg.org/Starfish/Imagepipe) <sup>**[[F-Droid](https://f-droid.org/en/packages/de.kaffeemitkoffein.imagepipe)]**</sup>
+- [**Trail Sense**](https://github.com/kylecorry31/Trail-Sense) <sup>**[[F-Droid](https://f-droid.org/en/packages/com.kylecorry.trail_sense)]**</sup>
```
Comment (offa) [inline README.md:254]:
```suggestion
- [**ImagePipe**](https://codeberg.org/Starfish/Imagepipe) <sup>**[[F-Droid](https://f-droid.org/app/de.kaffeemitkoffein.imagepipe)]**</sup>
- [**Trail Sense**](https://github.com/kylecorry31/Trail-Sense) <sup>**[[F-Droid](https://f-droid.org/app/com.kylecorry.trail_sense)]**</sup>
```

File: README.md [hunk 5]
```diff
@@ -268,6 +273,7 @@ Table of Contents
 - [**Music Player GO**](https://github.com/enricocid/Music-Player-GO) <sup>**[[F-Droid](https://f-droid.org/app/com.iven.musicplayergo)]**</sup>
 - [**Odeon**](https://github.com/thibseisel/android-odeon) <sup>**[[F-Droid](https://f-droid.org/app/fr.nihilus.music)]**</sup>
 - [**Metro**](https://github.com/MuntashirAkon/Metro) <sup>**[[F-Droid](https://f-droid.org/app/io.github.muntashirakon.Music)]**</sup>
+- [**Pulse Music**](https://github.com/HardcodeCoder/PulseMusic) <sup>**[[F-Droid](https://f-droid.org/en/packages/com.hardcodecoder.pulsemusic)]**</sup>
```
Comment (offa) [inline README.md:276]: Unfortunately Pulse Music is no longer developed, it has been removed therefore (see #154). :-(

**Comment by offa** [comment]
Thank you! :+1:  I have added some minor comments.

**Comment by CYB3R-G0D** [comment]
> Thank you! 👍 I have added some minor comments.

✌️

File: README.md (hunk 5 above)
Comment (CYB3R-G0D) [reply README.md:276]: Oops 😅

**Comment by CYB3R-G0D** [comment]
✌️

**Review by CYB3R-G0D — COMMENTED** [review]
👍

File: README.md (hunk 5 above)
Comment (offa) [reply README.md]: No problem :smile:

**Comment by offa** [comment]
Perfect!

**Pull request merged**